index.js

```diff
@@ -30,17 +30,55 @@ const defaults = {
   baseURL: '',
 };
 
-function addFile(builder, url, file, overrideHeadersOption) {
+function addAsset(
+  builder,
+  baseURL,
+  assetFullName, // Directory(optional) + assetShortName. Depending if called recursively or not.
+  assetShortName,
+  assetRelativeDirToBaseURL,
```

Comment:
Could you combine these three into `relativeAssetPath`?

Reply:
PTAL!